modules/preview-react/InformationHighlight/lib/parts/Body.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import {createComponent, ExtractProps} from '@workday/canvas-kit-react/common';
+import {createStencil, handleCsProp} from '@workday/canvas-kit-styling';
+import {base, system} from '@workday/canvas-tokens-web';
+import {Text} from '@workday/canvas-kit-react/text';
+
+const informationHighlightBodyStencil = createStencil({
+  base: () => {
+    return {
+      ...system.type.subtext.large,
+      color: base.blackPepper300,
+      gridColumn: '2',
+      margin: 0,
```

Comment:
```suggestion
      margin: `0 0 ${cssVar(system.space.x2) 0}`,
```
We want to add 8px bottom margin to the `body` to create some visual space between the body and the link

Reply:
it got real mad about the last 0, so i have `0 0 ${cssVar(system.space.x2)}` instead. what did the 0 in the curly brace do?

Reply:
You can remove that last curly:
`0 0 ${cssVar(system.space.x2) 0`

Reply:
Fat finger sorry about that.